app/views/CategorisationResultView.scala.html

```diff
@@ -14,14 +14,65 @@
  * limitations under the License.
  *@
 
+@import models.helper._
+
 @this(
         layout: templates.Layout,
-        govukButton: GovukButton
+        govukButton: GovukButton,
+        govukPanel: GovukPanel
 )
 
-@()(implicit request: Request[_], messages: Messages)
+@(scenario: Scenario)(implicit request: Request[_], messages: Messages)
+
+@panel = @{
+    val panelContent = scenario match {
+        case Category1NoExemptions => messages("categorisationResult.Category1NoExemptions.panelBody")
+        case Category3NoAssessments => messages("categorisationResult.Category3NoAssessments.panelBody")
+        case StandardNoSupplementaryUnits => messages("categorisationResult.StandardNoSupplementaryUnits.panelBody")
+        case Category1 => messages("categorisationResult.Category1.panelBody")
+        case Category2 => messages("categorisationResult.Category2.panelBody")
+    }
+    govukPanel(Panel(title = messages("categorisationResult.heading"), content = HtmlContent("<p class=\"govuk-body\" style=\"color:#FFFFFF\">" + panelContent + "</p>")))
```

Comment:
The colour style needs to be here. govukPanel is made to the design system spec, and the prototype did not follow that exactly. Though the design system says nothing against what has been done. Anyway, this needs to be here otherwise other styling is used. Will not remove unless a better option is provided that works. I've already tried a few things